docs/quickstart.md

```diff
@@ -0,0 +1,125 @@
+# Quickstart
+
+The following section provides the info regarding how to prepare and use
+`ralph-cli`.  After reading it, you'll be able to perform a successful scan of
+your iDRAC/iLO host, and save its result to Ralph (at this moment, it will be
+limited only to MAC addresses, but that is going to change soon!).
+
+## Getting ralph-cli binary
```

Comment:
Please separate development and end user installation methods. glide should not be known for the EU.